torch_tvm/compiler.cpp

```diff
@@ -176,13 +179,16 @@ tvm::relay::Function TVMCompiler::convertToRelay(
   auto output = tvm::relay::Tuple(n);
 
   tvm::Array<tvm::relay::Var> free_vars = tvm::relay::FreeVars(output);
-  AT_CHECK(
-      free_vars.size() <= input_vars.size(),
-      "Determined ",
-      free_vars.size(),
-      " free vars but only ",
-      input_vars.size(),
-      " inputs");
+  // Changing this to warning because layer norm may not make use of optional weight
```

Comment:
if the number of inputs is less than the number of free variables something is broken.  Any conditional on the use of an optional weight is either 1) constant or 2) dynamic.  If it's constant we need to propagate that information and reduce the number of free variables.  Otherwise we should create a dummy variable as an input

Reply:
@bwasti, so this was one thing I was vary of. I do want to preserve this however the way op is that is it has optional weight and bias and there is single relay op. I did not want to bloat code and add a separate relay op with weight and bias. I did see that in batch_norm this is handled differently, but I wanted to avoid adding another CallNode which I was not sure would be eliminated. If you have a suggestions I am open to it.

Reply:
rm this comment?